packages/default/scss/datetimepicker/_theme.scss

```diff
@@ -1,6 +1,21 @@
 @include exports( "datetimepicker/theme" ) {
 
     // Datetime picker
-    .k-datetimepicker {}
+    .k-datetimepicker {
+
+        //Disabled state
+        &:disabled,
+        &[disabled],
+        &.k-state-disabled,
```

Comment:
As of v5, we've switched from `k-state-*` to more generic  `k-*`. You've noticed that in some modules there are still selectors with `k-state-*` class. But they will be updated in the R2 2022 effort. 

In this PR we work with numeric and datetimepicker that was part of the v5 effort for R1 2022. So the `&.k-state-disabled` selector have to be removed.